### DISCUSSION TITLE
Error on windows

### DISCUSSION THREAD

**Discussion by McAllstar** [Koenkk/zigbee2mqtt#23050] [General]
anyone help me with this error? 
error:    zh:zstack:znp: Failed to determine if path is valid: 'Error: ENOENT: no such file or directory, lstat 'C:\COM5''

- I've verified in device manage that the sonoff usb 3.0 is recognised and assigned to com5
- I've tried the following in my configuration.yaml 
     serial:
  port: '\\\\.\\COM5'
and 
serial:
  port: '\\.\\COM5'

**Comment by Koenkk** [comment]
Can you try just COM5 (without slashes)

**Reply by quensen** [reply]
This didn’t work for me.

**Comment by quensen** [comment]
For me,
```
serial:
  port: /COM3
```
worked on Windows 11

**Comment by danielmmmm** [comment]
I get this error whether I use "COM20", "/COM20", or "\COM20" on Windows 10 Pro with zigbee2mqtt v1.39.1.

When i use "port: COM20", zigbee2mqtt will look for a folder COM20 in its own directory, "D:\Zigbee\zigbee2mqtt\COM20" in my case. With a forward or backward slash, it looks for "D:\COM20".

For some reason, it connects fine to my CC2652P Dongle regardless of the error.
